src/coprocessor/codec/batch/lazy_column.rs

```diff
@@ -553,7 +598,7 @@ mod benches {
         }
     }
 
-    /// Bench performance of push 10 bytes to a vector based LazyBatchColumn.
+    /// Bench performance of pushing 10 bytes to a vector based LazyBatchColumn.
```

Comment:
Could we move the bench test into the `benches` directory in another PR?

Reply:
But we are not introducing benchmarks in this PR?